### PR TITLE
Adds Carthage framework build

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: FBShimmering.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -236,6 +320,64 @@
...
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_TESTABILITY = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				INFOPLIST_FILE = Shimmering/Info.plist;
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				MTL_ENABLE_DEBUG_INFO = YES;
```
Comment (grp) [inline FBShimmering.xcodeproj/project.pbxproj]: A lot of these build settings don't seem relevant to Shimmer. Can you make sure only relevant ones are included?

File: FBShimmering.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -236,6 +320,64 @@
...
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_TESTABILITY = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				INFOPLIST_FILE = Shimmering/Info.plist;
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
+				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
```
Comment (grp) [inline FBShimmering.xcodeproj/project.pbxproj]: Shimmer should work as far back as iOS 7 (or maybe even earlier).

**Comment by grp** [comment]
I'm happy to take this one for Shimmer (it's simple enough there shouldn't be an issue), but it would be great to make it as simple as possible so it's the least Xcode goop to maintain. Thanks!

File: FBShimmering.xcodeproj/project.pbxproj (hunk 2 above)
Comment (flovilmart) [inline FBShimmering.xcodeproj/project.pbxproj]: The deployment target is obviously wrong here, but dynamic frameworks require 8.0.

**Comment by flovilmart** [comment]
@grp I'll improve that in the upcoming minutes

File: FBShimmering.xcodeproj/project.pbxproj (hunk 1 above)
Comment (flovilmart) [inline FBShimmering.xcodeproj/project.pbxproj]: Hi, those are the default build settings that Xcode provide for Framework targets, I'm not sure which ones can be removed. Still remove the MTL_ENABLE_DEBUG_INFO

**Comment by tbaranes** [comment]
@grp @flovilmart Any news about this PR?  I see there's conflicts, but what's the status, any chance that got merge soon?

**Comment by tbaranes** [comment]
Any chance to get this PR merge soon? I see that their conflicts now.
I ended up making my own fork with Carthage support, it's pretty similar to this one. Let me know if I can help :)
